setup.py

```diff
@@ -116,7 +118,7 @@ def run(self):
         "oss": oss,
         "ssh": ssh,
         # NOTE: https://github.com/inveniosoftware/troubleshooting/issues/1
-        ":python_version=='2.7'": ["futures", "pathlib2"],
+        ":python_version=='2.7'": ["futures", "pathlib"],
```

Comment:
You already added `pathlib2` to install_requires, do we need `pathlib` here as well?